web/ui/mantine-ui/src/pages/query/UPlotChart.tsx

```diff
@@ -95,7 +95,9 @@ const UPlotChart: FC<UPlotChartProps> = ({
         className={classes.uplotChart}
       />
       <Text fz="xs" c="dimmed" ml={40} mt={-25} mb="lg">
-        Click: hide single series, Ctrl+click: show single series
+        Click: show single series,{" "}
+        {navigator.userAgent.includes("Mac") ? "Cmd" : "Ctrl"} + click: hide
```

Comment:
It's more aesthetic ;) 
 
```suggestion
        {navigator.userAgent.includes("Mac") ? '⌘' : 'ctrl'} + click: hide
```

Reply:
Hah, changed :)